### DISCUSSION TITLE
Tab title instead horizontal tabs?

### DISCUSSION THREAD

**Discussion by qutebones** [piroor/treestyletab#3265] [Q&A] [answered]
I have removed the horizontal tabs and there is empty space with minimize/restore/close on the right, can I display the tab title there?

**Comment by irvinm** [comment]
This solution might still work ... been a little while.  https://github.com/piroor/treestyletab/issues/2387

You can basically "shift" those buttons down and it should get rid of the empty space.  (I used to use this solution)

**Comment by qutebones** [comment, marked as answer]
I don't want to get rid of the empty space, I want to use it for the tab title.

I have this:
![Screenshot 2022-12-20 161703](https://user-images.githubusercontent.com/36499184/208676736-f5dccc69-ca02-4d79-b98c-67226af1bc03.png)

Want like this:
![Tab tittle](https://user-images.githubusercontent.com/36499184/208676785-e90d5e73-2c0f-4233-acd6-a73c5c72f69a.png)

**Reply by irvinm** [reply]
Gotcha.  Just checking, but do you have "Title Bar" enabled in the customization section?  It is the title bar that has the tab title and I would think that hiding the tabs, but enabling the title bar might get you what you want.

![image](https://user-images.githubusercontent.com/979729/208679818-2bc2fa6b-3024-4c2b-b1bc-4db302f144ef.png)

**Reply by irvinm** [reply]
Scratch this as it won't work as you want out of the box.  I tried this and get the tab title and the buttons on the same row, but now there is a mostly blank row below that.  Maybe there is a way to hide that row and get you want you want, but would require some investigation on how to hide that row.

**Reply by qutebones** [reply]
This works. Thin strip with title and buttons.

![изображение](https://user-images.githubusercontent.com/36499184/208777582-fa0905a2-8202-4d4d-9e8d-c0c1488ad892.png)

Thank you.

**Reply by irvinm** [reply]
@qutebones will you mark this as solved so people know in the future?  Thx.

**Comment by irvinm** [comment]
How about this?

![image](https://user-images.githubusercontent.com/979729/208690289-71e3aa2f-8872-4a80-9b54-582b871b3680.png)

Just use:  https://github.com/MrOtherGuy/firefox-csshacks/blob/master/chrome/hide_tabs_toolbar.css

... and enable the title bar.

![image](https://user-images.githubusercontent.com/979729/208690830-f3a825b7-f387-4389-ba11-3575076b0615.png)
